content/community.md

```diff
@@ -12,16 +12,18 @@ Join the [GUAC public slack channel](https://openssf.slack.com/archives/C03U677Q
 
 ### Community Calls
 
-We host monthly community calls available for all to join.
+We host monthly community calls on the 3rd Thursday every month available for all to join.
 
-<a target="_blank" href="https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NTRsazR2cWUxaHVkYXVlOGt1dDNwZDBhNGdfMjAyMzAyMTZUMTgwMDAwWiBjXzg0ZjFmY2FhZGVhMmM0NTZlYTBkNWQ2OTljMzIwZWU5ZDc1NzY0ODQ0NzRlYmVmY2U1N2M0N2QxZWFlYjAyZDZAZw&tmsrc=c_84f1fcaadea2c456ea0d5d699c320ee9d7576484474ebefce57c47d1eaeb02d6%40group.calendar.google.com&scp=ALL"><img border="0" src="https://www.google.com/calendar/images/ext/gc_button1_en.gif"></a>
+<a target="_blank" href="https://calendar.google.com/calendar/embed?src=c_84f1fcaadea2c456ea0d5d699c320ee9d7576484474ebefce57c47d1eaeb02d6%40group.calendar.google.com"><img border="0" src="https://www.google.com/calendar/images/ext/gc_button1_en.gif"></a>
 
 ### Office Hours
 
 We host GUAC office hours where you can come by and ask questions about using
-GUAC or contributing to the project.
+GUAC or contributing to the project. We hold office hours every 2 weeks and
```

Comment:
confirming with poll

Reply:
looks like 9am is fine in community poll!